Dockerfile

```diff
@@ -1,94 +1,115 @@
+        libssl-dev \
+        make \
+        xz-utils \
+        zlib1g-dev' && \
+    apt-get update && \
+    apt-get install -y $buildDeps --no-install-recommends && \
+    rm -rf /var/lib/apt/lists/* && \
+    curl -fSL "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz" -o python.tar.xz && \
+    curl -fSL "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc" -o python.tar.xz.asc && \
+    export GNUPGHOME="$(mktemp -d)" && \
+    gpg --keyserver pgp.mit.edu --recv-keys "$GPG_KEY" && \
+    gpg --batch --verify python.tar.xz.asc python.tar.xz && \
+    rm -r "$GNUPGHOME" python.tar.xz.asc && \
+    mkdir -p /usr/src/python && \
+    tar -xJC /usr/src/python --strip-components=1 -f python.tar.xz && \
+    rm python.tar.xz && \
+    \
+    cd /usr/src/python && \
+    ./configure --enable-shared --enable-unicode=ucs4 && \
+    make -j$(nproc) && \
+    make install && \
+    ldconfig && \
+    pip3 install --no-cache-dir --upgrade --ignore-installed pip==$PYTHON_PIP_VERSION && \
+    pip3 install --no-cache-dir `grep 'numpy' /requirements.txt` && \
+    find /usr/local -depth \
+        \( \
+            \( -type d -a -name test -o -name tests \) \
+            -o \
+            \( -type f -a -name '*.pyc' -o -name '*.pyo' \) \
+        \) -exec rm -rf '{}' + && \
+    rm -rf /usr/src/python ~/.cache
 
-# Red Hat and Debian use different names for this file. git2R wants the latter.
-# See conda-recipes GH 423
-RUN ln -s /opt/conda/lib/libopenblas.so /opt/conda/lib/libblas.so && \
-    ln -s /opt/conda/lib/libopenblas.so /opt/conda/lib/liblapack.so && \
-    ln -s /opt/conda/lib/libssl.so /opt/conda/lib/libssl.so.6 && \
-    ln -s /opt/conda/lib/libcrypto.so /opt/conda/lib/libcrypto.so.6
+# Make some useful symlinks that are expected to exist.
+RUN cd /usr/local/bin && \
+    ln -s python3 python && \
+    ln -s python3-config python-config
 
-# Install boto in the base environment for private s3 channel support.
-# Install Python Packages
-COPY .condarc /opt/conda/.condarc
-COPY environment.yml environment.yml
-RUN conda install -y boto && \
-    conda install -y nomkl && \
-    conda env update -f environment.yml && \
-    conda clean --all -y
+# Now install the rest of the dependencies from the requirements file.
+RUN pip install --no-cache-dir -r /requirements.txt
+RUN rm -rf ~/.cache/*
 
 # We aren't running a GUI, so force matplotlib to use
 # the non-interactive "Agg" backend for graphics.
 ENV MATPLOTLIBRC=${HOME}/.config/matplotlib/matplotlibrc
-RUN mkdir -p ${HOME}/.config/matplotlib
-RUN echo "backend      : Agg" > ${HOME}/.config/matplotlib/matplotlibrc
+RUN mkdir -p `dirname ${MATPLOTLIBRC}`
+RUN echo "backend      : Agg" > ${MATPLOTLIBRC}
 
-# Run matplotlib once to build the font cache
+# Run matplotlib once to build the font cache.
 RUN python -c "import matplotlib.pyplot"
 
-ENV VERSION=2.0.1 \
-    VERSION_MAJOR=2 \
+ENV VERSION=3.0.0 \
```

Comment:
Oh, right. Conda is gone completely. That would be a breaking change for anyone who was conda-installing.